src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -101,13 +102,13 @@ public Content processPreview(EomFile eomFile, String transactionId) {
         UUID uuid = UUID.fromString(eomFile.getUuid());
 
         PublishEligibilityChecker eligibilityChecker =
-            PublishEligibilityChecker.forEomFile(eomFile, uuid, transactionId);
+                PublishEligibilityChecker.forEomFile(eomFile, uuid, transactionId);
```

Comment:
I just downloaded the latest and tried it. It's totally different from the old one. Most obvious is the fact that indentation has been changed to 2 spaces instead of 4. Looks weird IMHO. Gonna revert.